modules/4337/contracts/Safe4337Module.sol

```diff
@@ -212,9 +212,21 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
     }
 
     /**
-     * @dev Checks if the signatures length is correct and does not contain additional bytes. The function does not
+     * @notice Checks if the signatures length is correct and does not contain additional bytes. The function does not
      * check the integrity of the signature encoding, as this is expected to be checked by the {Safe} implementation
      * of {checkSignatures}.
+     * @dev Safe account has two types of signatures: EOA and Smart Contract signatures. While the EOA signature is
+     * fixed in size, the Smart Contract signature can be of arbitrary length. Safe encodes the Smart Contract
+     * signature length in the signature data. If appropriate length checks are not performed during the signature
```

Comment:
Can you move the first sentence further down to where it is relevant? I feel like this leading statement is completely unrelated to the following sentence and makes the documentation confusing for me :see_no_evil:.

Reply:
https://github.com/safe-global/safe-modules/pull/471/commits/3dff65b5656955f1b83d40b27641f72c5d864e97